core/controllers/library.py

```diff
@@ -73,6 +73,12 @@ class LibraryPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         search_mode = 'search' in self.request.url
+        rank_mode = 'explorations' in self.request.url
+
+        if search_mode:
```

Comment:
Well, I was thinking of making the 'minimal' change, which implies leaving it in the frontend, for no other reason than that it's currently in the frontend. At some point, it seems reasonable to pass the title (or, rather, the title i18n code) from the backend, just as we do for the group headers, but when we make that change I think we should do it fully and properly. Hence the suggestion to postpone it for now.